sol-did/scripts/createTestFixtureAccount.ts

```diff
@@ -88,7 +88,13 @@ export async function idlAddress(programId: PublicKey): Promise<PublicKey> {
 
   // Init account
 
-  await service.initialize().rpc();
+  const init = await service.initialize().instructions();
+  const migrate = await service.migrate().instructions();
+
+  console.log(`Init Inst: ${init[0].data.toString('hex')} instructions`);
+  console.log(`Migrate Inst: ${migrate[0].data.toString('hex')} instructions`);
+
+  return;
```

Comment:
Is this intentional ?

Reply:
Oh no thanks, that should not have gotten in there.